### DISCUSSION TITLE
Ruby 3.1.2 fails to compile with Xcode 14

### DISCUSSION THREAD

**Discussion by heyvaleria** [rbenv/ruby-build#2066] [Build failures]
### Steps to reproduce the behavior

```
asdf install ruby 3.1.2
```
Computer details:
MacBook Pro (16-inch, 2021) 
Chip Apple M1 Pro
Xcode 14.0.1

### Expected vs. actual behavior

```
It seems your ruby installation is missing psych (for YAML output).
```

### Logs

[Link to the gist with the full log](https://gist.github.com/heyvaleria/67aac1dc4592eef26b47ad0d40cb1598)

**Comment by hsbt** [comment]
It's a known issue. see https://bugs.ruby-lang.org/issues/18912
